packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-cards.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--gse-ui-selectorCard-simple-gap);
```

Comment:
Styling the host can leave the component exposed to unexpected overrides from the application css.
If possible move this styling into the shadow dom.